examples/computer_vision/detr_coco_pytorch/startup-hook.sh

```diff
@@ -0,0 +1,12 @@
+apt-get update
```

Comment:
Why do we need this in the startup hook?

Reply:
for some reason this is needed to install unzip.